GliaWidgets/View/Call/CallStyle.swift

```diff
@@ -44,6 +44,10 @@ public class CallStyle: EngagementStyle {
     /// Style of the call view bottom button bar (with buttons like "Chat", "Video", "Mute", "Speaker" and "Minimize").
     public var buttonBar: CallButtonBarStyle
 
+    /// The text shown in the top section of the view when visitor is put on hold
+    public var onHoldTopText: String?
+    public var onHoldBottomText: String?
```

Comment:
Why do we need to have these two strings optional? If these strings are nil visitors can miss context what happened with UI.

Reply:
this was done due to backward compatiblity. I saw that similar approach was used previously while developing new features. But default values are actually quite good alternative tht you proposed and I didn't think of it myself. WIll fix